tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "include": ["src", "types"],
+  "compilerOptions": {
+    "target": "es5",
+    "module": "esnext",
+    "lib": ["dom", "esnext"],
+    "importHelpers": true,
```

Comment:
I didn't add `tslib` to my `devDeps` but I do use `...rest` and `...spread` in some places... not sure if this can be removed or if I should add `tslib` or if this is fine 🤷‍♂

Reply:
There's an `_extends` helper added to the compiled code as well as code around rest/spread. I also checked and saw that `tslib` was in my `node_modules` already and it's actually [a dependency of `tsdx`](https://github.com/palmerhq/tsdx/blob/37d7664dd3953994f44761d490390c56e5048987/package.json#L68), so maybe that part of the default `package.json` it creates is outdated?

Reply:
Ehhh confused again because per the [`tslib` docs](https://github.com/microsoft/tslib), the output code should import `tslib` if this is true.... why isn't it??